src/expressions/if-expr.md

```diff
@@ -111,6 +111,27 @@ match EXPR {
 }
 ```
 
+If the expression is a [lazy boolean operator expression][_LazyBooleanOperatorExpression_] then
+this is a error as it is ambiguous which is problematic for the implementation
```

Comment:
I'd leave out the "which is problematic". Just state it's ambiguous with a planned future change in the language.

Reply:
Changed this.